pkg/actions/nodegroup/update.go

```diff
@@ -12,47 +12,51 @@ import (
 )
 
 func (m *Manager) Update() error {
-	ng := m.cfg.ManagedNodeGroups[0]
-	describeNodegroupOutput, err := m.ctl.Provider.EKS().DescribeNodegroup(&eks.DescribeNodegroupInput{
-		ClusterName:   &m.cfg.Metadata.Name,
-		NodegroupName: &ng.Name,
-	})
-
-	if err != nil {
-		if managed.IsNotFound(err) {
-			return fmt.Errorf("could not find managed nodegroup with name %q", ng.Name)
+	for _, ng := range m.cfg.ManagedNodeGroups {
```

Comment:
LGTM, just one small nit which is more of a personal preference:

can we move the contents of this `for` into another private func? simply because the process is quite long so it is easy to lose sight that this is all in a loop 🤷‍♀️

Reply:
Good idea! I split the contents into `updateNodegroup` and the strictly `updateConfig` bit into `updateUpdateConfig` (I know, cringe)

Reply:
😂  excellent